js/previewplugin.js

```diff
@@ -146,3 +146,43 @@ $(document).ready(function(){
 		}
 	}
 });
+
+(function () {
+
+	var SidebarPreview = function () {
+		this.preview = new OCA.Files_Markdown.Preview();
```

Comment:
I would never just copy-past a block of code and forget to properly clean things up